### PR TITLE
Add *LabelCluster specs

### PR DESCRIPTION
**Pull request by Apollon77**
This is another case of derived Clusters. LabelCluster is the base and FixedLabelCluster and UserLabelCluster the drived ones - also "slightly different" when it comes ro Writable/Non Writeable (and access level)

### CODE REVIEW

**Comment by Apollon77** [comment]
Ack, done

File: src/matter/cluster/LabelCluster.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+/**
+ * This cluster provides a feature to tag an endpoint with zero or more labels.
+ * Derived from LabelCluster ({@link MatterCoreSpecificationV1_0} § 9.7)
+ *
+ * clusterRevision: 1
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.9
+ */
+export const UserLabelCluster = Cluster({
+    /** Is a base cluster, so no id */
```
Comment (mfucci) [inline src/matter/cluster/LabelCluster.ts:33]: This comment is incorrect now... delete?

File: src/matter/cluster/LabelCluster.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+/**
+ * This cluster provides a feature for the device to tag an endpoint with zero or more read only labels.
+ * Derived from LabelCluster ({@link MatterCoreSpecificationV1_0} § 9.7)
+ *
+ * clusterRevision: 1
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.8
+ */
+export const FixedLabelCluster = Cluster({
+    /** Is a base cluster, so no id */
```
Comment (mfucci) [inline src/matter/cluster/LabelCluster.ts:53]: This comment is incorrect now... delete?

**Review by mfucci — DISMISSED** [review]

File: src/matter/cluster/LabelCluster.ts (hunk 1 above)
Comment (Apollon77) [reply src/matter/cluster/LabelCluster.ts]: copy/paste error ;-(

**Review by mfucci — APPROVED** [review]

**Pull request merged**